share/dkg/pedersen/dkg_test.go

```diff
@@ -667,6 +667,10 @@ func TestDKGResharing(t *testing.T) {
 	newSecret, err := share.RecoverSecret(suite, newSShares, thr, defaultN)
 	require.NoError(t, err)
 	require.Equal(t, oldSecret.String(), newSecret.String())
+	// 3.
+	for i := 0; i < len(dkgs); i++ {
+		require.Equal(t, shares[i].Public(), newShares[i].Public())
```

Comment:
the overall DKG public key should remain constant, not the public key of each share, right ?

Reply:
That's correct, but the `Public()` method returns the public key of the distributed key:

https://github.com/dedis/kyber/blob/a008b2d45c5c8675d2f01d140b7232166757a013/share/dkg/pedersen/structs.go#L24

So this should pass.